demo/demo.less

```diff
@@ -383,6 +387,10 @@ html, body {
   padding: 0 16px;
 }
 
+.app-header .mdl-layout__drawer-button .material-icons:hover {
+  transform: rotate(20deg);
```

Comment:
Can you please give a before/after screenshot of this?  I am not sure what this does, and your PR description didn't help me figure it out.

Reply:
These changes rotates the icon when hover
before
![o1](https://user-images.githubusercontent.com/58463645/115385161-6fd19e00-a1f5-11eb-93e9-55b841c404b9.PNG)
after
![n1](https://user-images.githubusercontent.com/58463645/115385184-7829d900-a1f5-11eb-9f7c-063402152875.PNG)
actually its difficult to find changes through images as its motion effect.

its like settings icon on YouTube player when clicked
![y1](https://user-images.githubusercontent.com/58463645/115385650-ff774c80-a1f5-11eb-8a18-5de40005ae55.PNG)

Reply:
I played around with this change, and I found that there are two things I dislike about it:

1. 20 degrees looks weird.  It's a six-sided shape, so intervals of 30 degrees look better than 20 degrees.  Otherwise, it just looks crooked.
2. Having no transition animation makes the change snap to the rotation instantly, which doesn't provide such a pleasing effect.

Having played around with the settings, I think this looks really nice:

```css
.app-header .mdl-layout__drawer-button .material-icons {
  &:hover {
    /* Spin the gear icon in the settings menu when hovered. */
    transform: rotate(90deg);
  }

  /* Animate the spin. */
  transition: transform 0.5s ease-in-out;
}
```

What do you think?

Reply:
I think we should change time to 0.3s  that even looks better to me!

Reply:
Sounds good to me.  Thanks!